v2/internal/controllers/samples_test.go

```diff
@@ -78,7 +79,7 @@ func runGroupTest(tc *testcommon.KubePerTestContext, groupVersionPath string) {
 
 	rg := tc.NewTestResourceGroup()
 	useRandomName := !testcommon.IsFolderExcluded(groupVersionPath, randomNameExclusions)
-	samples, err := testcommon.NewSamplesTester(tc.NoSpaceNamer, tc.GetScheme(), groupVersionPath, tc.Namespace, useRandomName, rg.Name).LoadSamples()
+	samples, err := testcommon.NewSamplesTester(tc.NoSpaceNamer, tc.GetScheme(), groupVersionPath, tc.Namespace, useRandomName, rg.Name, tc.AzureSubscription).LoadSamples()
```

Comment:
Maybe it's time to format this call with one parameter per line.
```suggestion
	samples, err := testcommon.NewSamplesTester(
		tc.NoSpaceNamer, 
		tc.GetScheme(), 
		groupVersionPath, 
		tc.Namespace,
		 useRandomName, 
		 rg.Name, 
		 tc.AzureSubscription).
		 LoadSamples()
```